src/ui/public/kbn_share/kbn_share.less

```diff
@@ -0,0 +1,42 @@
+.sharing-ui-menu__List {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+.sharing-ui-menu__ListWrapper,
+.sharing-ui-menu__SharePanel {
```

Comment:
Per Zoom convo:

- We'll use component names as namespaces (e.g. `shareMenu`) instead of abbreviated prefixes (e.g. `sui-`) because they're a little more concrete.
- We'll use indentation to indicate parent-child relationships.
- We'll change the pattern of state classes to use the component name first (I'll update the CSS style guide to reflect this).